site/.eslintrc.yaml

```diff
@@ -75,6 +75,8 @@ rules:
   "@typescript-eslint/restrict-template-expressions": "off"
   # TODO: Investigate whether to enable this rule & fix and/or disable all its complaints
   "@typescript-eslint/unbound-method": "off"
+  # TODO: Investigate whether to enable this rule & fix and/or disable all its complaints
+  "@typescript-eslint/no-redundant-type-constituents": "off"
```

Comment:
Adding this exception reduced the lint error from 46 to 6

Reply:
This looks like to be a good role tho. https://typescript-eslint.io/rules/no-redundant-type-constituents/

Reply:
Done :)